backend/restApi.js

```diff
@@ -22,10 +22,21 @@ router.use(jsonParser);
 
 router.post("/crossword", (req, res) => {
   const crossword = req.body;
-  if (req.body.adminToken !== process.env.ADMIN_TOKEN) {
-    res.status = 401;
-    res.send("Access denied");
-    return;
+  // If ADMIN_TOKEN is set (via command line), use it
+  if (process.env.ADMIN_TOKEN) {
+    if (req.body.adminToken !== process.env.ADMIN_TOKEN) {
```

Comment:
Nesting not needed, can be done in one if statement

Reply:
I pushed a new commit fixing the issues in a nice way!